app/controllers/admin/routes.py

```diff
@@ -143,6 +142,26 @@ def addRecurringEvents():
     recurringEvents = calculateRecurringEventFrequency(preprocessEventData(request.form.copy()))
     return json.dumps(recurringEvents, default=str)
 
+@admin_bp.route('/courseProposals', methods=['GET'])
```

Comment:
Should do a check to make sure that the user is a faculty.  I think a specific route with the user's username would also be ideal.  Similar to the volunteer profile.

Reply:
Oops, misunderstood this, but it still needs a check to see if the user is an admin.  I was able to get to this page and withdraw a course as a student through putting in the URL